$OpenBSD: patch-src_widgets_styles_qgtk2painter_cpp,v 1.1.1.1 2016/12/25 14:13:11 zhuk Exp $
Fix Qt5 in situation GTK+ shared library version is overridden.
--- src/widgets/styles/qgtk2painter.cpp.orig	Wed May 25 18:46:17 2016
+++ src/widgets/styles/qgtk2painter.cpp	Sun Jul 10 14:15:13 2016
@@ -96,8 +96,7 @@ static void initGtk()
 #ifndef QT_NO_LIBRARY
     static bool initialized = false;
     if (!initialized) {
-        // enforce the "0" suffix, so we'll open libgtk-x11-2.0.so.0
-        QLibrary libgtk(QLS("gtk-x11-2.0"), 0, 0);
+        QLibrary libgtk(QLS("gtk-x11-2.0"));
 
         QGtk2PainterPrivate::gdk_pixmap_new = (Ptr_gdk_pixmap_new)libgtk.resolve("gdk_pixmap_new");
         QGtk2PainterPrivate::gdk_pixbuf_get_from_drawable = (Ptr_gdk_pixbuf_get_from_drawable)libgtk.resolve("gdk_pixbuf_get_from_drawable");
